Msgraph email xsup 55352 (#41448)
demisto/content: Packs/ApiModules/ReleaseNotes/2_3_1.md

#### Scripts

##### MicrosoftGraphMailApiModule

> - Improved the **Send Email** command to include a retry mechanism when sending emails with attachments, ensuring more reliable delivery in case of temporary failures.

Reviewer: ```suggestion
- Improved the **Send Email** command to include a retry mechanism for emails with attachments, ensuring more reliable delivery in case of temporary failures.
```
